api/openapi-spec/v3/apis__apiextensions.k8s.io__v1_openapi.json

```diff
@@ -774,7 +774,7 @@
             "type": "string"
           },
           "reason": {
-            "description": "reason provides a machine-readable validation failure reason that is returned to the caller when a request fails this validation rule. The HTTP status code returned to the caller will match the reason of the reason of the first failed validation rule. The currently supported reasons are: \"FieldValueInvalid\", \"FieldValueForbidden\", \"FieldValueRequired\", \"FieldValueDuplicate\". If not set, default to use \"FieldValueInvalid\". All future added reasons must be accepted by clients when reading this value.",
+            "description": "reason provides a machine-readable validation failure reason that is returned to the caller when a request fails this validation rule. The HTTP status code returned to the caller will match the reason of the reason of the first failed validation rule. The currently supported reasons are: \"Invalid\", \"Forbidden\", \"NotFound\", \"AlreadyExists\". If not set, default to use \"Invalid\". All future added reasons must be accepted by clients when reading this value.",
```

Comment:
we actually do want to keep the same values (FieldValueInvalid, etc) since the value set here gets returned as a field error, which is user-visible